pkg/provider/aws/resources/eks.go

```diff
@@ -79,6 +95,15 @@ func CreateEksCluster(appName string, clusterName string, subnets []*Subnet, sec
 	dag.AddResource(cluster)
 	dag.AddDependency(cluster, clusterRole)
 
+	err := cluster.createFargateLogging(references, dag)
+	if err != nil {
+		zap.S().Warnf("Unable to set up Fargate logging manifests for cluster %s: %s", clusterName, err.Error())
```

Comment:
instead of (or in addition to, your pick) just printing the err, also add it as a zap field (`zap.Error(err)` or something like that, I forget exactly). That will let us get the full stack trace on telemetry.

ditto below

Reply:
well wouldnt we actually fail compilation with zap.Error? The reason i opted for warn is because these charts just affect streaming logs to CW and wouldnt affect any bit of the functionality

Reply:
I'm not saying log at error level, I'm just saying attach the field. When we send the telemetry, we look at the fields, and any that are of type `error` we extract the full stack trace from and send that.